maven-embedder/src/main/java/org/apache/maven/cli/MavenCli.java

```diff
@@ -1593,7 +1622,10 @@ private TransferListener determineTransferListener( final boolean quiet,
                                                         final CommandLine commandLine,
                                                         final MavenExecutionRequest request )
     {
-        if ( quiet || commandLine.hasOption( CLIManager.NO_TRANSFER_PROGRESS ) )
+        String ciEnv = request.getSystemProperties().getProperty( "env.CI" );
+        boolean quietCI = "true".equals( ciEnv ) && !commandLine.hasOption( FORCE_INTERACTIVE );
```

Comment:
`!"false".equals( ciEnv )`, value can be anything (`CI=<platform>`) so guess only testing negative case is saner on the long run

Reply:
This is not the way the value is defined.

Reply:
> Should not but you know, real life. So just testing `!=null && !false` is more robust as of today IMHO.

I'd like to see what other tools do.

Reply:
https://github.com/gradle/gradle/blob/master/build-logic/buildquality/src/test/kotlin/gradlebuild/testcleanup/TestFilesCleanupServiceTest.kt#L31
https://github.com/gradle/gradle/blob/master/gradle/shared-with-buildSrc/mirrors.settings.gradle.kts#L46
https://github.com/gradle/gradle/blob/master/subprojects/docs/src/snippets/buildCache/conditional-action/groovy/build.gradle#L6

Reply:
Well, when I search for "CI=true" on all GH I get different results.

Reply:
Are there CI servers that are actually setting it to another value than true that you know of? 
Anyway, it doesn't hurt I guess, I have changed it